### PR TITLE
Bug 1852928 - Update review quality check bottom sheet header

### PR DESCRIPTION
**Pull request by MozillaNoah**
image

image

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1852928

### CODE REVIEW

File: fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt [hunk 1]
```diff
@@ -81,22 +85,54 @@ fun ReviewQualityCheckScaffold(
...
+    val betaBorderColor: Color
+    val betaTextColor: Color
+    if (isSystemInDarkTheme()) {
+        betaBorderColor = PhotonColors.LightGrey10
+        betaTextColor = FirefoxTheme.colors.textActionPrimary
+    } else {
+        betaBorderColor = FirefoxTheme.colors.actionTertiary
+        betaTextColor = FirefoxTheme.colors.textSecondary
+    }
+
```
Comment (MozillaNoah) [inline fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt:97]: I confirmed with Kate that the text & border colors here are one-off and intentionally do not have a matched-up color token.

Comment (rahulsainani) [reply fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt:97]: Should we create a bug to track this, so going forward we can align?

**Review by rahulsainani — APPROVED** [review]
Looks good! Left a comment about non theme colors.

File: fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt (hunk 1 above)
Comment (MozillaNoah) [reply fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt:97]: Not in this scenario. This won't be conforming to an Acorn token. There may be, long term, "one-off" tokens we make at a feature scope for Fenix-only that are provided by the figma designer (i.e. not from the Acorn system), but those are very very early conversations.

**Pull request merged**